<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerMultiDimensionBrowserModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerMultiDimensionBrowserModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Active browser node:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_ActiveBrowser">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLMultiDimensionBrowserNode</string>
        </stringlist>
       </property>
       <property name="baseName">
        <string>MultiDimensionBrowser</string>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="InputSection">
     <property name="text">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="label_ParameterName">
        <property name="text">
         <string>Parameter name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="slider_ParameterValue">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ParameterValue">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ParameterUnit">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_MultiDimensionInput">
        <property name="text">
         <string>Multi-dimension dataset:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_MultiDimensionRoot">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLHierarchyNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>true</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="OutputSection">
     <property name="text">
      <string>Output</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_VirtualOutput">
        <property name="text">
         <string>Virtual output node:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="MRMLNodeComboBox_VirtualOutput">
        <property name="toolTip">
         <string>This hierarchy node will contain the nodes corresponding to the selected parameter value</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLHierarchyNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>VirtualOutput</string>
        </property>
        <property name="addEnabled">
         <bool>true</bool>
        </property>
        <property name="removeEnabled">
         <bool>true</bool>
        </property>
        <property name="renameEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerMultiDimensionBrowserModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_MultiDimensionRoot</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerMultiDimensionBrowserModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_VirtualOutput</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerMultiDimensionBrowserModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>MRMLNodeComboBox_ActiveBrowser</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
